cloudinit/user_data.py

```diff
@@ -361,10 +403,18 @@ def is_skippable(part):
     return False
 
 
-# Coverts a raw string into a mime message
-def convert_string(raw_data, content_type=NOT_MULTIPART_TYPE):
-    """convert a string (more likely bytes) or a message into
-    a mime message."""
+def message_from_string(string) -> Message:
+    return email.message_from_string(string)
+
+
+def convert_string(
+    raw_data: Union[str, bytes], content_type=NOT_MULTIPART_TYPE
```

Comment:
None of our call sites set the kwarg, could we make this a local variable instead?